### PR TITLE
Some top-level options are now only on sub-commands

### PR DESCRIPTION
**Pull request by dbeatty10**
## What are you changing in this pull request and why?

Context: https://github.com/dbt-labs/dbt-core/issues/7490

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.

### CODE REVIEW

File: website/docs/guides/migration/versions/02-upgrading-to-v1.5.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ The following env vars have been renamed, for consistency with the convention fo
 
 As described in [dbt-core#7169](https://github.com/dbt-labs/dbt-core/pull/7169), command-line parameters that could be silent before will no longer be silent. See [dbt-labs/dbt-core#7158](https://github.com/dbt-labs/dbt-core/issues/7158) and [dbt-labs/dbt-core#6800](https://github.com/dbt-labs/dbt-core/issues/6800) for more examples of the behavior we are fixing.
 
+Some options that could previously be specified before a sub-command now can only be specified afterwards. e.g. `dbt --profiles-dir . run` is no longer valid and must be `dbt run --profiles-dir .` instead.
```
Comment (mirnawong1) [inline website/docs/guides/migration/versions/02-upgrading-to-v1.5.md]:
```suggestion
Some options that could previously be specified before a sub-command can now only be specified afterward. For example, `dbt --profiles-dir . run` isn't valid anymore, and instead, you need to use `dbt run --profiles-dir .`
```

**Review by mirnawong1 — DISMISSED** [review]
hey @dbeatty10 thank you for opening this up! have a tiny non blocker sugg - approved for you either way!

**Comment by dbeatty10** [comment]
Thank you @mirnawong1 !

Accepted your suggestion, updated out-of-date branch, and enabled auto-merge so we'll see this merge after CI checks.

**Review by mirnawong1 — APPROVED** [review]
🚀 looks good to me @dbeatty10 🙏

**Pull request merged**